cliquet/events.py

```diff
@@ -72,21 +94,35 @@ def on_new_request(event):
         if hasattr(event.request, 'parent'):
             return
         current = transaction.get()
-        current.addAfterCommitHook(_notify_resource_events,
+        current.addBeforeCommitHook(_notify_resource_events_before,
+                                    args=(event.request,))
+        current.addAfterCommitHook(_notify_resource_events_after,
                                    args=(event.request,))
 
     config.add_subscriber(on_new_request, NewRequest)
 
 
-def get_resource_events(request):
+def get_resource_events(request, after=False):
```

Comment:
``after`` is not really self-descripting here. Not sure about a better naming. What about ``after_commit`` or something that conveys the fact that the action really happened?